### PR TITLE
SOT-131:  Nova folding scheme

### PR DESCRIPTION
**Pull request by VanhGer**
# Check List

- [ ] Did you check your grammar / reader proof your document using ChatGPT?
- [ ] Don't forget to squash commits into meaningful chunks before merging
- [ ] After merging this, please update your task status!
- [ ] Did you verify that your math formulas are rendered correctly?

### CODE REVIEW

File: terms/ivc.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+IVC helps to break a long computation into a series of small identical
+steps that significantly reduces the prover memory requirements.
+
+## Core Idea of IVC
+
+With initial input $z_0$, an IVC schema allows a prover to produce a proof $\Pi _ i$ 
+for the statement $z _ i = F ^ {(i)}(z _ 0)$ given a proof $\Pi _ {i-1}$ for the
+statement $z _ {i-1} = F ^ {(i-1)} (z _ 0)$.
+
+Informally, at step $n$, prove computes a new state $z_n$ and a proof $\Pi_n$
```
Comment (hduoc2003) [inline terms/ivc.md:17]:
```suggestion
Informally, at step $n$, prove computes a new statement $z_n$ and a proof $\Pi_n$
```

File: terms/ivc.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+that proves prover has a witness $(z _ {n-1}, w _ n, \Pi _ {n-1})$  such that $z_n$ is correct
+relative to the previous state $z _ {n-1}$ and the proof $\Pi _ {n-1}$ is correct relative
+to the previous state $z _ {i-1}$.
```
Comment (hduoc2003) [inline terms/ivc.md:20]:
```suggestion
to the previous state $z _ {n-1}$.
```

**Review by hduoc2003 — APPROVED** [review]

File: terms/folding_scheme.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Folding Scheme
+
```
Comment (bjergsen243) [inline terms/folding_scheme.md:2]: I think you need to write about what this part is

**Review by bjergsen243 — APPROVED** [review]

**Review by ktokto313 — APPROVED** [review]

File: terms/folding_scheme.md (hunk 3 above)
Comment (sonntuet1997) [reply terms/folding_scheme.md:2]: Add the definition of a folding scheme

Comment (VanhGer) [reply terms/folding_scheme.md]: I will add in the next commit.

**Pull request merged**